### ISSUE TITLE
License

### ISSUE THREAD

**Issue by N-Olbert** [d1820/CodeDocumentor#24]
Whats the license of this code / extension?

**Comment by d1820** [comment]
You're free to use it however you like, you're free to fork it and adapt it
on your own, you're also free to make modifications and submit PRs it's
basically unlicensed free for use.

if you look at the read me at the bottom I do call out a big thanks to the
original creator where I forked it from them. I did the modifications with
their permission because they were not willing to continue making
modifications to their version of the documenter..

so basically go crazy have fun use it however you want

This may have been sent with voice to text, so I apologize for any
misspellings or incorrect wording
